projects/storyteller/webapp/tslint.json

```diff
@@ -0,0 +1,131 @@
+{
+  "rulesDirectory": [
+    "codelyzer"
+  ],
+  "rules": {
```

Comment:
It's `tslint` rules, description on how should be written typescript code.
And good thing is it's not just declaration. `tslint` can fix errors automatically.

To see errors:
```
ng lint
```

To fix errors:
```
ng lint --fix
```
(or `ng lint -fix`, if you have earlier version of ng)

If you don't have ng:
```
npm install -g @angular/cli
```

Reply:
BTW, tslint can fix `end without new line` in `.ts` files.

Reply:
You can see here what each rule means:
https://palantir.github.io/tslint/rules/

Reply:
thanks a lot Vadim!!!